package.json

```diff
@@ -30,6 +30,30 @@
     "./src/*.js": "./src/*.js"
   },
   "main": "dist/jquery.js",
+  "scripts": {
+    "build": "npm install && npm run build-all-variants",
+    "build-all-variants": "grunt custom:slim --esm --filename=jquery.slim.module.js && grunt custom --esm --filename=jquery.module.js && grunt custom:slim --filename=jquery.slim.js && grunt custom",
+    "jenkins": "npm run test:browserless",
+    "lint:dev": "eslint .",
+    "lint:dist": "eslint --no-ignore dist/jquery.js dist/jquery.slim.js dist-module/jquery.module.js dist-module/jquery.slim.module.js",
+    "lint:json": "jsonlint --quiet package.json",
+    "lint:cache": "concurrently -r \"npm:lint:dev -- --cache\" \"npm:lint:json\"",
+    "lint": "concurrently -r \"npm:lint:dev\" \"npm:lint:json\" && npm run lint:dist",
+    "prepare": "husky install",
+    "start": "grunt watch",
+    "test:browserless": "grunt && npm run lint && npm run test:node_smoke_tests && grunt test:slow",
```

Comment:
This is equivalent to what we have now so it’s fine for now but on GitHub Actions we should only run lint on a single Mode version. Otherwise, if a PR has a linting error, an inline comment will be added by GitHub to each violation the number of times we run the lint. This is not a pleasant view. 😄

Reply:
Yeah, that's a good idea.

If you do that, please make sure it's still executed on `npm test`. `npm test` should run all the checks & tests we have. We don't run it in CI directly anywhere but I sometimes use it to verify my changes locally before creating a PR.